install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-VAMP_INSTALLER_VERSION=1.13.2
+VAMP_INSTALLER_VERSION=2.0.1
```

Comment:
Does this version contain Dmitry’s changes related to k8s permissions? If yes, are we ready to push it?

Reply:
Yes I discussed it with Gordon

Reply:
ok 👍